### PR TITLE
Add Python and .NET code samples for OpenTelemetry

### PR DESCRIPTION
**Pull request by tushar-datadog**
### Merge instructions

- [ ] Please merge after reviewing

### CODE REVIEW

**Review by brett0000FF — APPROVED** [review]

**Comment by brett0000FF** [comment]
@tushar-datadog - LGTM! Shoot me a ping when you want this merged. 🚀

File: content/en/opentelemetry/interoperability/instrumentation_libraries.md [hunk 1]
```diff
@@ -309,6 +349,52 @@ To use OpenTelemetry instrumentation libraries with the Datadog .NET SDK:
 1. Set the `DD_TRACE_OTEL_ENABLED` environment variable to `true`.
 2. Follow the steps to configure each library, if any, to generate OpenTelemetry-compatible instrumentation via `ActivitySource`
 
+The following example demonstrates how to instrument the `Hangfire` OpenTelemetry integrations with the Datadog .NET SDK:
+
+```js
```
Comment (bm1549) [inline content/en/opentelemetry/interoperability/instrumentation_libraries.md:354]: I think GitHub supports C# highlighting

https://github.com/github-linguist/linguist/blob/main/lib/linguist/languages.yml

File: content/en/opentelemetry/interoperability/instrumentation_libraries.md [hunk 2]
```diff
@@ -309,6 +349,52 @@ To use OpenTelemetry instrumentation libraries with the Datadog .NET SDK:
...
+cusing System;
```
Comment (bm1549) [inline content/en/opentelemetry/interoperability/instrumentation_libraries.md:355]: This looks like a typo
```suggestion
using System;
```

**Review by bm1549 — APPROVED** [review]
LGTM assuming this code compiles, runs, and produces correct output on your laptop

File: content/en/opentelemetry/interoperability/instrumentation_libraries.md (hunk 1 above)
Comment (brett0000FF) [reply content/en/opentelemetry/interoperability/instrumentation_libraries.md:354]: Good point. You can use `csharp`.

File: content/en/opentelemetry/interoperability/instrumentation_libraries.md [hunk 3]
```diff
@@ -309,6 +349,52 @@ To use OpenTelemetry instrumentation libraries with the Datadog .NET SDK:
...
+using OpenTelemetry.Trace;
+using OpenTelemetry.Resources;
+using OpenTelemetry.Instrumentation.Hangfire;
+using OpenTelemetry;
+
+class Program
+{
+    static void Main(string[] args)
+    {
+        // Create a new TracerSettings object and configure it
```
Comment (zacharycmontoya) [inline content/en/opentelemetry/interoperability/instrumentation_libraries.md:367]:
```suggestion
        // Create an OpenTelemetry TracerProvider to initialize the OpenTelemetry Hangfire instrumentation and build the configuration
```

File: content/en/opentelemetry/interoperability/instrumentation_libraries.md [hunk 4]
```diff
@@ -309,6 +349,52 @@ To use OpenTelemetry instrumentation libraries with the Datadog .NET SDK:
...
+        var openTelemetry = Sdk.CreateTracerProviderBuilder()
+            .SetResourceBuilder(ResourceBuilder.CreateDefault().AddService("hangfire-demo2"))
+            .AddHangfireInstrumentation()
```
[the remark this answers is not in the document]
Comment (zacharycmontoya) [reply content/en/opentelemetry/interoperability/instrumentation_libraries.md:370]: I just added a suggestion above which may suffice

File: content/en/opentelemetry/interoperability/instrumentation_libraries.md [hunk 5]
```diff
@@ -309,6 +349,52 @@ To use OpenTelemetry instrumentation libraries with the Datadog .NET SDK:
...
 
+The following example demonstrates how to instrument the `Hangfire` OpenTelemetry integrations with the Datadog .NET SDK:
+
+```csharp
+using System;
+using Hangfire;
+using Hangfire.MemoryStorage;
+using OpenTelemetry.Trace;
+using OpenTelemetry.Resources;
+using OpenTelemetry.Instrumentation.Hangfire;
```
Comment (bouwkast) [inline content/en/opentelemetry/interoperability/instrumentation_libraries.md:360]: Just wondering if we should include the packages as well here?

```
  <ItemGroup>
    <PackageReference Include="Hangfire" Version="1.8.14" />
    <PackageReference Include="Hangfire.MemoryStorage" Version="1.8.1.1" />
    <PackageReference Include="OpenTelemetry" Version="1.9.0" />
    <PackageReference Include="OpenTelemetry.Api" Version="1.9.0" />
    <PackageReference Include="OpenTelemetry.Instrumentation.Hangfire" Version="1.9.0-beta.1" />
  </ItemGroup>
```

**Review by bouwkast — APPROVED** [review]
I copied the .NET code and ran it and looks to be good thanks!

File: content/en/opentelemetry/interoperability/instrumentation_libraries.md (hunk 5 above)
Comment (zacharycmontoya) [reply content/en/opentelemetry/interoperability/instrumentation_libraries.md:360]: +1

Comment (tushar-datadog) [reply content/en/opentelemetry/interoperability/instrumentation_libraries.md:360]: So I wanted to stay in parity with how we show code snippets elsewhere. We dont include the dlls that are imported as part of it. I think snippet will suffice for now to atleast help the user get started. 

I will create a separate PR explaining the dependencies that are present and how the OTel SDK is not one of them

**Pull request merged**